README.md

```diff
@@ -39,6 +39,20 @@ import a11yAudit from 'ember-a11y-testing/test-support/audit';
 
 // ...elided for brevity
 
+test('Some test case', async function(assert) {
+  visit('/');
+  await a11yAudit();
+  assert.ok(true, 'no a11y errors found!'));
+});
+```
+
+If you are using Ember 2.18 or below, you will need to use the the andThen() helper and a regular function (as opposed to `async function`).
```

Comment:
Hmm, you can use `await` going back many versions. It shouldn’t be tied to 2.18.

Reply:
Also, shouldn't `visit()` use `await`?

Reply:
Thanks, I'll try and update both of these this week. @rwjblue I believe the only pre-req to using this syntax is using `ember-cli-qunit` 4.2? I think that became default in `ember-cli` 2.18 but anyone using older version can update manually. Do I have that correct?

Reply:
ember-cli-qunit@4.2 allowed using the newer testing styles proposed in [emberjs/rfcs#268](https://github.com/emberjs/rfcs/blob/master/text/0268-acceptance-testing-refactor.md) and [emberjs/rfcs#232](https://github.com/emberjs/rfcs/blob/master/text/0232-simplify-qunit-testing-api.md), however you could still use `await visit(...)` using the (now legacy) `moduleForAcceptance` API's as long as your application allows async/await in general.

Reply:
Items updated. Sorry for the delay. A new baby since the previous commit severely cut into free time